clang/runtime/dpct-rt/include/dpct/math.hpp

```diff
@@ -2055,6 +2055,98 @@ class joint_matrix {
   matrix_accessor x;
   const size_t num_elements;
 };
+
+/// Loads 1 8x8 b16 matrix from local memory to private memory (32-bits per wi)
```

Comment:
```suggestion
/// Loads 1 8x8 b16 matrix from local memory to private memory (32-bits per work item)
```

Reply:
Please give more details on how the loaded data will be distributed on private memory.   is it a work-group function or sub-group function?  Make sure user can use it after reading the comments.